modules/ROOT/pages/apikit-graphql-api-implementation.adoc

```diff
@@ -0,0 +1,525 @@
+           "user": {
+               "username": "tblackmore15"
+           },
+           "title": "GraphQL by example",
+           "body": "Let's talk about GraphQL",
+           "comments": []
+       }
+   }
+}
+----
+
+=== Add a Comment to the New Post
+
+This use case posts a comment in reply to the post created in the previous step. It also asks for the service to return the body of the comment created and the original post with its list of comments to check that the new one is included.
+
+==== Make a Query
+
+----
+mutation PostComment($userId: ID!, $postId: ID!, $body: String!) {
+   postComment(userId: $userId, postId: $postId, body: $body) {
+       id
+       body
+       post {
+           id
+           title
+           comments {
+               id
+               user {
+                   username
+               }
+               body
+           }
+       }
+   }
+}
+----
+
+==== Make Variables
+
+----
+{
+   "userId": 34,
+   "postId": 101,
+   "body": "Nice post! Thanks for sharing! It's just what I was looking for"
+}
+----
+
+==== Trigger a Request
+
+----
+curl --location --request POST 'http://localhost:8081/graphql' \
+--header 'Content-Type: application/json' \
+--data-raw '{"query":"mutation PostComment($userId:ID!,$postId:ID!,$body:String!){postComment(userId:$userId,postId:$postId,body:$body){id body post{id title comments{id user{username}body}}}}","variables":{"userId":34,"postId":101,"body":"Nice post! Thanks for sharing! It'\''s just what I was looking for"}}'
+----
+
+==== Get a Response
+
+----
+{
+   "data": {
+       "postComment": {
+           "id": "201",
+           "body": "Nice post! Thanks for sharing! It's just what I was looking for",
+           "post": {
+               "id": "101",
+               "title": "GraphQL by example",
+               "comments": [
+                   {
+                       "id": "201",
+                       "user": {
+                           "username": "ekosex"
+                       },
+                       "body": "Nice post! Thanks for sharing! It's just what I was looking for"
+                   }
+               ]
+           }
+       }
+   }
+}
+----
+
+=== Reply to the New Comment in the Post
+
+This example request adds a comment of the user that created the original post in response to the previous comment. The query asks the service to return all the information of the post in addition to the new reply.
+
+==== Make a Query
+
+----
+mutation ReplyToComment($commentId: ID!, $userId: ID!, $body: String!) {
+   replyToComment(commentId: $commentId, userId: $userId, body: $body) {
+       id
+       body
+       post {
+           id
+           title
+           comments {
+               id
+               body
+               replies {
+                   id
+                   user {
+                       username
+                   }
+                   body
+               }
+           }
+       }  
+   }
+}
+----
+
+==== Make Variables
```

Comment:
Look at all these `====` and make sure they're consistently singular or plural. I think they're usually singular.

Reply:
Great catch! Thanks!